Jenkinsfile.release.prepare

```
@@ -108,14 +110,14 @@ pipeline {
                         def branchConfig = [ dependent_repositories : repositories]
 
                         echo "Write dsl branch_config => ${branchConfig}"
-                        sh 'rm dsl/branch_config.yaml'
-                        writeYaml file: 'dsl/branch_config.yaml', data: branchConfig, overwrite: true
+                        sh "rm ${branchConfigFile}"
```

```suggestion
                        sh "rm -f ${branchConfigFile}"
```
`-f` in case it does not exist.

---

```suggestion
                        sh "rm -f ${branchConfigFile}"
```